blocks/library/quote/index.js

```diff
@@ -7,7 +7,7 @@ import Editable from 'components/editable';
 
 const { parse, html, query } = hpq;
 
-const fromValueToParagraphs = ( value ) => value.map( ( paragraph ) => `<p>${ paragraph }</p>` ).join( '' );
+const fromValueToParagraphs = ( value ) => value ? value.map( ( paragraph ) => `<p>${ paragraph }</p>` ).join( '' ) : '';
```

Comment:
Is this fixing a bug? Do we really want to support an undefined `value`? Do we need some concept of default attributes, whether that's an explicit property of a block or even just how we destructure `attributes` in `edit` and `save`?